### PR TITLE
BaSP-TG-74:Misalign employees form fixed

### PR DESCRIPTION
**Pull request by aletiscornia**
New title added to the form and fixed form style.

### CODE REVIEW

File: src/Components/Employees/employees.module.css [hunk 1]
```diff
@@ -14,8 +14,14 @@
   background-color: #f7f7f7a4;
 }
 
-.title {
+.titleform {
```
Comment (ignacioaristo) [inline src/Components/Employees/employees.module.css:17]: lets use camelCase convention.

File: src/Components/Employees/employees.module.css [hunk 2]
```diff
@@ -57,14 +63,6 @@
...
 
-.addform {
-  display: flex;
-  flex-direction: column;
-  justify-items: center;
-  align-items: center;
-  margin-bottom: 40px;
-}
-
 .formcontrol {
```
Comment (ignacioaristo) [inline src/Components/Employees/employees.module.css:66]: Same here

**Review by ignacioaristo — DISMISSED** [review]

File: src/Components/Employees/employees.module.css (hunk 1 above)
Comment (aletiscornia) [reply src/Components/Employees/employees.module.css:17]: fixed

File: src/Components/Employees/employees.module.css (hunk 2 above)
Comment (aletiscornia) [reply src/Components/Employees/employees.module.css:66]: fixed

**Review by GonzaloAsencio — APPROVED** [review]

**Comment by Ignacio-Sormani** [comment]
@sofihoj @javierperezraffo I'll approve the PR. We still need to align some styles, but the form is repositioned. So we can move forward with the merge and improve styles later

**Review by Ignacio-Sormani — APPROVED** [review]

**Review by javierperezraffo — APPROVED** [review]

**Pull request merged**